ppdet/data/transform/operators.py

```diff
@@ -1169,15 +1188,28 @@ def apply_brightness(self, img):
         img += delta
         return img
 
+    def _blend(self, alpha, img, img_mean):
+        img *= alpha
+        img_mean *= (1 - alpha)
+        img += img_mean
+
     def __call__(self, sample, context=None):
         img = sample['image']
+        img_mean = None
         if self.random_apply:
-            distortions = np.random.permutation([
+            functions = [
                 self.apply_brightness, self.apply_contrast,
-                self.apply_saturation, self.apply_hue
-            ])
+                self.apply_saturation
+            ]
+            if not self.corner_jitter:
+                functions.append(self.apply_hue)
```

Comment:
You can the arg `self.hue`  to judge whether use `apply_hue` instead of  `corner_jitter`

Reply:
Done